guava/src/com/google/common/math/BigIntegerMath.java

```diff
@@ -144,11 +148,12 @@ public static int log2(BigInteger x, RoundingMode mode) {
    *     is not a power of ten
    */
   @GwtIncompatible // TODO
-  @SuppressWarnings("fallthrough")
-  public static int log10(BigInteger x, RoundingMode mode) {
+  @SuppressWarnings({"fallthrough","lowerbound:argument.type.incompatible"// if x is posItive, `x.longValue()` returns positive
```

Comment:
The important property here isn't that `x` is positive (which the checker does know). It's that if `x` fits in a `long`, then `longValue` is polymorphic over `x`'s value with respect to the lowerbound type system. I suggest changing the explanation to make that clear.